From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 2 Mar 2018 11:55:16 +0100
Subject: [PATCH 09/19] logind: voidify a function we never check the return
 value of

(cherry picked from commit a3ddf73c0e4abf8e3c1b8fd91eac469220a5b44b)
---
 src/login/logind-session-device.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/login/logind-session-device.c b/src/login/logind-session-device.c
index 65b4bb8..706245d 100644
--- a/src/login/logind-session-device.c
+++ b/src/login/logind-session-device.c
@@ -106,7 +106,7 @@ static int session_device_notify(SessionDevice *sd, enum SessionDeviceNotificati
         return sd_bus_send(sd->session->manager->bus, m, NULL);
 }
 
-static int sd_eviocrevoke(int fd) {
+static void sd_eviocrevoke(int fd) {
         static bool warned = false;
 
         assert(fd >= 0);
@@ -118,8 +118,6 @@ static int sd_eviocrevoke(int fd) {
                         warned = true;
                 }
         }
-
-        return 0;
 }
 
 static int sd_drmsetmaster(int fd) {
